Data/Sys/GameSettings/GMPE01.ini

```diff
@@ -282,7 +358,7 @@ C205BEF8 00000004
 60000000 00000000
 *50 turns becomes 99 due to integer cap.
 
-$Mechanics - Event Accessibility Doesn't Depend on Mini/Mega Status [Celerizer]
+$Mechanics - Event Accessibility Doesnt Depend on Mini/Mega Status [Celerizer]
```

Comment:
I noticed you removed `'` from several code titles (Boo's > Boos) does this break the cheat parser or is this a mistake?

Reply:
Neither my IDE must of auto formatted it. I can fix it manually.

Reply:
Personally I'd prefer to keep the `'` if it works.